addon/-private/utils/parse-response-headers.js

```diff
@@ -25,7 +25,10 @@ export default function parseResponseHeaders(headersString) {
       continue;
     }
 
-    let field = header.substring(0, j).trim();
+    let field = header
+      .substring(0, j)
+      .trim()
+      .toLowerCase();
```

Comment:
Would you also mind adding the original "unlowercased" field name to the headers object so anyone's app that is relying the the current behavior continues to work?